apps/docs/assets/styles/main.css

```
@@ -307,13 +156,6 @@ article > .cbd-container {
   }
 }
 
-.heading-wrapper {
-  position: relative;
-  padding-inline-start: 20px;
-  margin-inline-start: -20px;
-
-}
-
 article > h1 {
   padding-inline: 16px;
   padding-block-start: 40px;
```

```suggestion
  padding-block-start: 20px;
```
there's too much extra space at the top

---

unrelated to this scope